config/rubocop/1.9.1/rubocop.default.yml

```diff
@@ -589,7 +589,7 @@ Layout/EndAlignment:
   # situations, `end` should still be aligned with the keyword.
   # The value `start_of_line` means that `end` should be aligned with the start
   # of the line which the matching keyword appears on.
-  EnforcedStyleAlignWith: keyword
+  EnforcedStyleAlignWith: start_of_line
```

Comment:
We shouldn't be modifying the default, this is the base template **without** our customizations. Read [this comment](https://github.com/bespokepost/code-climate-config/pull/8#discussion_r628427101)

Make the changes in [this file](https://github.com/bespokepost/code-climate-config/blob/83a2ffb77b2fe6bfea6efcedd74b07e8bdae6b97/config/rubocop/1.9.1/rubocop-2.7.yml) instead.

Reply:
Actually seems that I read in the past by the looks of the 👍  I left there 😄 .